jbang-catalog.json

```diff
@@ -0,0 +1,10 @@
+{
+  "catalogs": {},
+  "aliases": {
+    "jfxcentral": {
```

Comment:
you can change this alias to anything, i.e. "run" then it will just be `jbang run@dlemmermann/jfxcentral`

I would recommend creating a `dlemmermann/jbang-catalog` since then it gets even shorter (`jbang jfxcentral@dlemmermann`) and a bit more decoupled.